modules/oe_content_entity/README.md

```diff
@@ -1,16 +1,80 @@
 # OpenEuropa corporate content entities
 
 This module and its sub-modules ship a set of corporate content entity types that are used to collect information
-of different classes on corporate content types.
+of different content classes on corporate content types. In order to use a specific corporate content entity type enable the related module.
```

Comment:
Remove `In order to use a specific corporate content entity type enable the related module.` as it's right below.